README.md

```diff
@@ -9,6 +9,8 @@ poor and doing so doesn't address a meaningful threat model.
 [![Documentation](https://docs.rs/rustls-pemfile/badge.svg)](https://docs.rs/rustls-pemfile/)
 
 # Release history
+- 1.0.4 (TBD)
```

Comment:
Can you set this to today's date? I can publish it afterwards.

Reply:
I've added the date in.